examples/keras_io/tensorflow/vision/mlp_image_classification.py

```diff
@@ -76,15 +75,11 @@ def build_classifier(blocks, positional_encoding=False):
     # Augment data.
     augmented = data_augmentation(inputs)
     # Create patches.
-    patches = Patches(patch_size, num_patches)(augmented)
+    patches = Patches(patch_size)(augmented)
     # Encode patches to generate a [batch_size, num_patches, embedding_dim] tensor.
     x = layers.Dense(units=embedding_dim)(patches)
     if positional_encoding:
-        positions = tf.range(start=0, limit=num_patches, delta=1)
-        position_embedding = layers.Embedding(
-            input_dim=num_patches, output_dim=embedding_dim
-        )(positions)
-        x = x + position_embedding
+        x = x + PositionEmbedding(sequence_length=num_patches)(x)
```

Comment:
This change fixed the run on Torch backend, and made the embedding appear in the summary of the model.